lib/transport/selenium-webdriver/browserstack.js

```diff
@@ -41,19 +58,25 @@ class Browserstack extends Selenium {
     this.settings.webdriver.start_process = false;
     this.settings.webdriver.port = Number(this.settings.webdriver.port);
 
-    if (!this.accessKey && process.env.BROWSERSTACK_KEY) {
-      this.settings.desiredCapabilities['browserstack.key'] = process.env.BROWSERSTACK_KEY;
+    const browserstackKey = process.env.BROWSERSTACK_ACCESS_KEY ?
+      process.env.BROWSERSTACK_ACCESS_KEY :
+      process.env.BROWSERSTACK_KEY;
+    if (!this.accessKey && browserstackKey) {
+      this.settings.desiredCapabilities['browserstack.key'] = browserstackKey;
     }
 
-    if (!this.username && process.env.BROWSERSTACK_USER) {
-      this.settings.desiredCapabilities['browserstack.user'] = process.env.BROWSERSTACK_USER;
+    const browserstackUsername = process.env.BROWSERSTACK_USERNAME ?
+      process.env.BROWSERSTACK_USERNAME :
+      process.env.BROWSERSTACK_USER;
+    if (!this.username && browserstackUsername) {
+      this.settings.desiredCapabilities['browserstack.user'] = browserstackUsername;
```

Comment:
Not needed

Reply:
please refer to [this comment](https://github.com/nightwatchjs/nightwatch/pull/3148/files/04591a03316194b805ad9ea1dfce18a933009453#r849253208).

Reply:
Please remove additional process envs those are not needed like `process.env.BROWSERSTACK_USERNAME`